admin/controllers/school.js

```diff
@@ -22,6 +18,7 @@ const schoolDataService = require('../services/data-access/school.data.service')
 const generatePinsService = require('../services/generate-pins.service')
 const { sortRecords } = require('../utils')
 const sortingAttributesService = require('../services/sorting-attributes.service')
+const checkDataService = require('../services/data-access/check.data.service')
```

Comment:
@pris54 @GeorgeChatzigiannis  This controller depends upon services **and** data-services.  It uses the data-services to pass data to the services!!!  This is a dependency nightmare.  We need to refactor this out going forward.